insights/specs/datasources/corosync.py

```diff
@@ -3,13 +3,14 @@
 """
 import os
 
-from insights.components.rhel_version import IsRhel7, IsRhel8, IsRhel9
+from insights.components.rhel_version import IsRhel7, IsRhel6
+from insights.combiners.redhat_release import RedHatRelease
 from insights.core.context import HostContext
 from insights.core.exceptions import SkipComponent
 from insights.core.plugins import datasource
 
 
-@datasource(HostContext, [IsRhel7, IsRhel8, IsRhel9])
+@datasource(HostContext, [IsRhel6, IsRhel7, RedHatRelease])
```

Comment:
As both `IsRhel*` and `RedHatRelease` are used here, and the `IsRhel*` is also depending on `RedHatRelease`, how about just use `RedHatRelease` only?